arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3579,50 +3579,57 @@ arangodb::Result lockDBServerTransactions(std::string const& backupId,
   }
 
   // Perform the requests
-  cc->performRequests(requests, lockWait + 1.0, Logger::BACKUP, false, false);
+  cc->performRequests(requests, lockWait + 5.0, Logger::BACKUP, false, false);
```

Comment:
The fix seems to be here! Maybe you could add some kind of explanation why the time has to be increased.

Reply:
No, this is not the fix at all. 1.0 should be good enough in all but a handful of cases. I just changed it to 5.0 since it does essentially cost nothing and widens the margin of error, for example for slow networks or overload.